pallets/collator-selection/src/migrations.rs

```diff
@@ -87,7 +88,7 @@ impl<T: Config> Pallet<T> {
 
 	pub fn post_migrate_v0_to_v1() -> Result<(), &'static str> {
 		use frame_support::migration::{have_storage_value, storage_key_iter};
-		if !(Self::on_chain_storage_version() == 1) {
+		if Self::on_chain_storage_version() != 1 {
```

Comment:
I think this migration code should be deleted. @Garandor

Reply:
This migration code has been applied to Calamari and is no longer in use. It is being preserved as 
- migration history
- Reference template for future migrations

This code shouldn't need to be touched, so unless you're trying to clear clippy warnings or the like, please just drop this change. If the change helps to keep our build logs cleaner, feel free to merge it as is.

I'm marking this as resolved for now. Feel free to reopen if it needs more discussion @bhgomes

Reply:
Yeah, I was just clearing `clippy` warnings.